tests/s3/test_object_tagging.py

```diff
@@ -1511,6 +1444,7 @@ def test_multipartupload_max_tags_42774(self):
         self.log.info("Retrieved tag of an object")
         self.log.info("Verify Multipart Upload with max no.of tags for an Object")
 
+    @pytest.mark.parallel
```

Comment:
Let's not introduce this marker, unless you have verified parallel execution of the tests

Reply:
Removed.